charts/rqlite/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: rqlite
+version: 0.1.0
+appVersion: 8.13.2
+description: The lightweight, distributed relational database built on SQLite 
+type: application
+home: https://rqlite.io/
+sources:
+  - https://github.com/rqlite/rqlite
```

Comment:
Source of the database, the chart, or should we just have a link to the GitHub org? (https://github.com/rqlite)

Reply:
It's really just informational for the chart user.  Could include links to the helm-charts repo and rqlite repos specifically, as well as the org link.  I can update to include all of the above.